go/adbc/driver/bigquery/connection.go

```diff
@@ -574,6 +576,10 @@ func (c *connectionImpl) newClient(ctx context.Context) error {
 			return err
 		}
 
+		if c.location != "" {
+			client.Location = c.location
```

Comment:
This is where it actually gets passed to the BQ SDK.

Reply:
Is this set to anything to begin with? What happens if we don't set it?